moveit_planners/stomp/include/stomp_moveit/cost_functions.hpp

```diff
@@ -86,46 +86,60 @@ CostFn getCostFunctionFromStateValidator(const StateValidatorFn& state_validator
     // Subsequent invalid states are assumed to have the same cause, so we are keeping track
     // of "invalid windows" which are used for smoothing out the costs per violation cause
     // with a gaussian, penalizing neighboring valid states as well.
-    for (int timestep = 0; timestep < values.cols() - 1; ++timestep)
+    for (int timestep = 0; timestep < values.cols(); ++timestep)
```

Comment:
Should timestep be renamed to something like sample_way-point?

Reply:
It's really just the timestep in the STOMP trajectory, though. It doesn't contain joint values